cli/cmd/verify.go

```diff
@@ -83,18 +79,12 @@ func runVerify(cmd *cobra.Command, _ []string) error {
 		filelist[fmt.Sprintf("manifest.%d.json", i)] = m
 	}
 	for _, p := range resp.Policies {
-		sha256sum := sha256.Sum256(p)
-		pHash := manifest.NewHexString(sha256sum[:])
-		gzipReader, err := gzip.NewReader(bytes.NewReader(p))
+		initdata := initdata.Raw(p)
+		digest, err := initdata.Digest()
 		if err != nil {
-			return fmt.Errorf("creating gzip reader: %w", err)
+			return fmt.Errorf("calculating initdata digest: %w", err)
 		}
-		policyDecompressed, err := io.ReadAll(gzipReader)
-		if err != nil {
-			return fmt.Errorf("decompressing policy: %w", err)
-		}
-		gzipReader.Close()
-		filelist[fmt.Sprintf("policy.%s.rego", pHash)] = policyDecompressed
+		filelist[fmt.Sprintf("initdata.%x.toml", digest)] = initdata
```

Comment:
IMO this is a regression in the user interface. There is really no way I can audit the initdata without further steps:

<img width="750" height="126" alt="image" src="https://github.com/user-attachments/assets/6729c1bc-3ca2-4fdb-981d-83916c67efda" />

If the policy could at least be printed as multi-line toml string, that would be much better.